README.md

```diff
@@ -10,6 +10,8 @@ This project is about to give me a handle as a student on how to create a web si
 
 ## Live demo
 
+https://ghislainmitahi.github.io/conference_cap/
```

Comment:
[OPTIONAL] Please, add a screenshot of the project to your readme.

Reply:
Sounds Good